interface/wx/grid.h

```diff
@@ -4336,6 +4356,8 @@ class wxGrid : public wxScrolledWindow
     */
     bool InsertRows(int pos = 0, int numRows = 1, bool updateLabels = true);
 
+    virtual void Refresh(bool eraseb = true, const wxRect* rect = NULL);
```

Comment:
This is not really needed, is it? It is already documented by `wxWindow`, so if absolutely necessary, it should mention what is different in this implementation.